.github/workflows/docker_production.yml

```diff
@@ -3,7 +3,7 @@ name: Docker-Production
 on:
   push:
     tags:
-      - desmos-morpheus-apollo-1-v*
+      - persistence-1-v*
```

Comment:
For the tags, our current naming convention is [chain name] + [chain id] + [v*]
So whenever we setup a new chain UI, we frontend need to check hasura first to find out the chain id.

You can find it in genesis:
`query MyQuery {
  genesis {
    chain_id
  }
}`

Reply:
done